src/app/chat/component/chat-page/chat-page.component.ts

```
@@ -0,0 +1,179 @@
+          this.selectedChat.time = time;
+          this.newMessage = '';
+        },
+        error: (err) => {
+          console.error('Failed to send message:', err);
+        }
+      });
+  }
+
+  onFileSelected(event: Event): void {
+    const input = event.target as HTMLInputElement;
+    if (input.files && input.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)